Tests/scripts/prepare_content_packs_for_testing.sh

```diff
@@ -37,10 +37,6 @@ echo "Updating modified content packs in the bucket ..."
 
 if [ ! -n "${NIGHTLY}" ]; then
     CONTENT_PACKS_TO_INSTALL_FILE="./Tests/content_packs_to_install.txt"
-
-  if [ -n "${CONTRIB_BRANCH}" ]; then
```

Comment:
why did you remove this?
it was added because the `Collect tests and packs to install` did not recognize it as a changed pack (as this might be a new pack and not part of the repo)

Reply:
Cause this would mistakenly add the contributor's pack name to the `content_packs_to_install.txt` file even when the contributor's pack wasn't pulled (which was the case in our build). The proper way to do this is to pull in the contributor's changes and have those changes detected in the `collect_test_list_and_content_packs` step the same way we do for all of our content.